ember-cli/2017-08/august-24.md

```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-08-24
+
+## Attendees
+
+
+## Agenda
```

Comment:
To comment on the above, I want to parse templates to an AST and analyse what icons are used. The use that information to remove (or avoid including) unused stuff.

There is two approaches: 
1) A two pass approach. I parse the templates eagerly before the build starts, and based no that I remove stuff (perhaps with some custom broccoli plugins using `postprocessTree`)
2) Reuse an existing AST transform. This sounds better because it avoids parsing and transversing templates twice, but by the time this is done I have no hook (I think) to remove unused stuff.